uefi/src/proto/console/draw_target.rs

```diff
@@ -0,0 +1,54 @@
+use embedded_graphics_core::prelude::{DrawTarget, OriginDimensions, Size, PixelColor, Pixel, IntoStorage};
+
+use super::gop::GraphicsOutput;
+
+// FIXME: offer conversions from C to current pixel color format?
+struct GraphicsDisplay<C: PixelColor> {
+    color: C,
```

Comment:
nit: Although this is not consistent across the code-base, personally I prefer `Box<dyn PixelColor` for all non-performance critical stuff. Having a generic parameter has the consequence that all wrapping structs (that library users may create in higher levels) are also forced to be generic. But this is just a nit and you don't have to address it.